MAINTENANCE.md

```diff
@@ -0,0 +1,59 @@
+
+This organization of issues into milestones is central to generating the module changelogs found on [hapi.dev](https://hapi.dev), and to creating transparency around what is contained in each release.
+
+#### Release Notes
+It is encouraged to create an issue with release notes for major changes, especially for modules that are intended to be used externally to the organization (e.g. hapi, bell, nes) versus hapi core (e.g. pez, topo, shot).  The [hapijs/hapi](https://github.com/hapijs/hapi) repository should have release notes for every major version of the framework.  [This](https://github.com/hapijs/hapi/issues/4017) is a great example of typical release note format and ideal level of detail.
+
+### Assignees
+The collaborator responsible for final review and merge of pull requests should mark themselves as the "assignee."  This leaves an at-a-glance paper trail primarily for convenience ("who understands the changes made in #365?"), and is in recognition of that person's involvement/contribution.  When a collaborator closes an issue that they were responsible for resolving (e.g. answered a user's support question), they may also mark themselves as the assignee.
+
+### Closing
+> _This list is not meant to indicate anything about the numerous other cases that may occur, it is meant only to offer some guidelines._
+
+ - Support issues without any response should remain open for two weeks then may be closed.
+ - If a support issue doesn't contain a sufficient reproduction it may be closed immediately with a comment.
+ - Any effort at resolving a support issue makes it eligible for being closed.  It's useful to point folks to the hapi hour slack for further support.  It is welcome for discussion to continue on the closed issue.
+ - Any issue that doesn't follow the relevant issue template may be closed immediately, optionally with a comment indicating to re-file the issue following the template.
+ - Feature requests that will not be satisfied may be closed with comment.
+ - Bugs should remain open until resolved.
+ - Release notes should be staged prior to publish and closed upon publish.
+ - PRs that will not be accepted may be closed with a comment.  If a PR is replaced, the new PR should be referenced in a comment.
+
+## Releases
+When a new version of a module is ready to be published, we adhere to the following release process:
```

Comment:
This section is clearly targeted at current maintainers, making it more of a guide than a policy. Can we change it to a more generic language to better match the other sections?

Reply:
I addressed this and your other feedback around the release section.  I left it as a guide, but I tried to make the language a bit more generic.  I also added a short introduction summarizing what a release is and noting the important policies that relate to the integrity of our releases.